tmt/steps/execute/__init__.py

```diff
@@ -337,13 +337,13 @@ def check_custom_results(self, test: "tmt.Test") -> List["tmt.Result"]:
             tmt.steps.execute.TEST_DATA,
             'results.yaml')
 
-        # Nothing to do if there's no result file
         if not os.path.exists(custom_results_path):
-            raise tmt.utils.FileError
+            raise tmt.utils.ExecuteError(
+                f"Custom results file '{custom_results_path}' not found.")
```

Comment:
No one complained against "Stating obvious but if no results.yaml is found or is in wrong format (or any other issue with it) the test (/foo in this case) ends as 'error'." (stated in the [issue](https://github.com/teemtee/tmt/issues/1074#issuecomment-1231421067)) so tmt should behave this way.

Raising tmt.utils.ExecuteError still makes tmt to exit before processing rest of the tests and report step is omitted. 
IMO that is something we should avoid and exit tmt only for not recoverable issues (e.g. machine got broken).

I believe correct thing to do is to 
```python
return [tmt.Result(data={'note': f"Custom results file '{custom_results_path}' not found.", 'result': 'error'}, test=test)
```

Or nicer note for the end user (among lines of tests error as it should provided custom result but it didn't)

Reply:
Thank you very much. I resolved it this way.